s3-backed-core/src/main/java/com/bakdata/kafka/AbstractS3BackedConfig.java

```diff
@@ -67,12 +69,25 @@
             + " with " + S3_ENDPOINT_CONFIG + ". Leave empty if default S3 region should be used.";
     public static final String S3_REGION_DEFAULT = "";
     public static final String S3_ACCESS_KEY_CONFIG = PREFIX + "access.key";
-    public static final String S3_ACCESS_KEY_DOC =
-            "AWS access key to use for connecting to S3. Leave empty if AWS credential provider chain should be used.";
+    public static final String S3_ACCESS_KEY_DOC = "AWS access key to use for connecting to S3. Leave empty if AWS"
+            + " credential provider chain or STS Assume Role provider should be used.";
     public static final String S3_ACCESS_KEY_DEFAULT = "";
     public static final String S3_SECRET_KEY_CONFIG = PREFIX + "secret.key";
-    public static final String S3_SECRET_KEY_DOC =
-            "AWS secret key to use for connecting to S3. Leave empty if AWS credential provider chain should be used.";
+    public static final String S3_SECRET_KEY_DOC = "AWS secret key to use for connecting to S3. Leave empty if AWS"
+            + " credential provider chain or STS Assume Role provider should be used.";
+    public static final String S3_ROLE_EXTERNAL_ID_CONFIG = "sts.role.external.id";
```

Comment:
Unless I'm missing something, the prefix for the added configs is missing 

```suggestion
    public static final String S3_ROLE_EXTERNAL_ID_CONFIG = PREFIX + "sts.role.external.id";
```